pkg/util/limitrange/limitrange.go

```diff
@@ -53,31 +55,25 @@ func addToSummary(summary, item corev1.LimitRangeItem) corev1.LimitRangeItem {
 	return summary
 }
 
-func violateMaxMessage(path *field.Path, keys ...string) string {
-	return fmt.Sprintf("the requests of %s[%s] exceeds the limits", path.String(), strings.Join(keys, ", "))
-}
-func violateMinMessage(path *field.Path, keys ...string) string {
-	return fmt.Sprintf("the requests of %s[%s] are less than the limits", path.String(), strings.Join(keys, ", "))
-}
-
-func (s Summary) validatePodSpecContainers(containers []corev1.Container, path *field.Path) []string {
+func (s Summary) validatePodSpecContainers(containers []corev1.Container, path *field.Path) field.ErrorList {
 	containerRange, found := s[corev1.LimitTypeContainer]
 	if !found {
 		return nil
 	}
-	reasons := []string{}
+	var allErrs field.ErrorList
```

Comment:
This conflicted with https://github.com/kubernetes-sigs/kueue/blob/06d4c763ce03891dc0f869e42d5557f9831f1682/pkg/util/limitrange/limitrange.go#L64